File: src/mappings/swig/ruby/test/test_emboss.rb

require 'biolib/emboss'

ENV['DATA'] = "../../../../test/data/emboss/"
trnTable = Biolib::Emboss.ajTrnNewI(1);
ajpseq   = Biolib::Emboss.ajSeqNewNameC("AGCT","Test sequence")
ajpseqt  = Biolib::Emboss.ajTrnSeqOrig(trnTable,ajpseq,1)
aa       = Biolib::Emboss.ajSeqGetSeqCopyC(ajpseqt)
raise 'Test failed '+aa if aa != 'SX'

exit 0

File: src/mappings/swig/perl/test/test_emboss.pl
BEGIN{unshift(@INC, "../emboss")};
use biolib::emboss;

# my $result = example::my_mod(7,3);
# print "my_mod returns $result!\n";

$ENV{'DATA'} = "../../../../test/data/emboss/";
$trnTable = emboss::ajTrnNewI(1);
$ajpseq   = emboss::ajSeqNewNameC("AGCT","Test sequence");
$ajpseqt  = emboss::ajTrnSeqOrig($trnTable,$ajpseq,1);
$aa       = emboss::ajSeqGetSeqCopyC($ajpseqt);

exit ($result != "SX");
